app/models/github_adapter.rb

```diff
@@ -0,0 +1,66 @@
+class GithubAdapter
+  def initialize(post)
+    @post = post
+  end
+
+  def create_pr
+    new_branch_sha = create_branch(master_sha)
```

Comment:
won't this have the same value as the `branch_sha` method? I find it confusing to have a branch_sha, and a new_branch_sha, when they seem to be the same thing